db/migrate/20160621204706_rename_stil_id_on_user.rb

```diff
@@ -0,0 +1,5 @@
+class RenameStilIdOnUser < ActiveRecord::Migration
+  def change
+    rename_column(:users, :stil_id, :student_id)
```

Comment:
> I do not feel that we need stil_id anymore, if you think the transition is smoother by removing and creating a new column I'm up for it 😄

👍 I think it will make the transition easier and faster